test/primitives.jl

```diff
@@ -58,13 +58,16 @@
     @test normal(p) == Vec(0, 0, 1)
     @test isnothing(boundary(p))
 
-    p = Plane(P3(0, 0, 0), V3(0, 0, 1))
-    @test p(T(1), T(0)) == P3(1, 0, 0)
-    @test p(T(0), T(1)) == P3(0, 1, 0)
+    n = V3(1, 1, 0)
+    p = Plane(P3(0, 0, 0), n)
+    @test abs(p.u⋅p.v) < eps(T)
+    @test abs(p.u⋅n) < eps(T)
+    @test abs(p.v⋅n) < eps(T)
```

Comment:
Notice that we cannot access fields of structs directly because that is not part of the public API. You can use methods of plane to recover the origin and normal vector instead.